nltk/twitter/twitter_demo.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+# Natural Language Toolkit: Twitter client
+#
+# Copyright (C) 2001-2015 NLTK Project
+# Author: Ewan Klein <ewan@inf.ed.ac.uk>
+# URL: <http://nltk.org/>
+# For license information, see LICENSE.TXT
+
+"""
+Examples to demo the :py:mod:`twitterclient` code.
```

Comment:
Over the last couple of years I've been trying to migrate demo code into module docstrings, for better accessibility. Is there any chance of doing that here? It might mean putting some utility functions in util.py, a common module in many NLTK packages.